webpack.config.js

```diff
@@ -4,27 +4,28 @@ var ExtractTextPlugin = require("extract-text-webpack-plugin");
 var WebpackAssetsManifest = require('webpack-assets-manifest');
 
 if (process.env.NODE_ENV === 'production') {
-  var jsFilename = "js/[name].[chunkhash].js",
-      cssFilename = "css/[name].[contenthash].css";
+  var jsFilename = "[name].[chunkhash].js",
+      cssFilename = "[name].[contenthash].css";
 
 }
 else {
-  var jsFilename = "js/[name].js",
-      cssFilename = "css/[name].css";
+  var jsFilename = "[name].js",
+      cssFilename = "[name].css";
 }
 
 var webpackConfig = {
   entry: {
-    'bootstrap': 'bootstrap',
-    'cart': './js/app/cart/index.jsx',
-    'homepage': './js/app/homepage/index.js',
-    'order-payment': './js/app/order/payment.js',
-    'order-tracking': [ 'whatwg-fetch', './js/app/order/tracking.jsx' ],
-    'profile-deliveries': './js/app/profile/deliveries.js',
-    'restaurant-form': './js/app/restaurant/form.jsx',
-    'delivery-form': './js/app/delivery/form.jsx',
-    'styles': './assets/css/main.scss',
-    'tracking': [ './assets/css/tracking.scss', './js/app/tracking/index.jsx' ]
+    'css/styles': './assets/css/main.scss',
+    'css/tracking': './assets/css/tracking.scss'
+    'js/bootstrap': 'bootstrap',
+    'js/cart': './js/app/cart/index.jsx',
+    'js/homepage': './js/app/homepage/index.js',
+    'js/order-payment': './js/app/order/payment.js',
+    'js/order-tracking': [ 'whatwg-fetch', './js/app/order/tracking.jsx' ],
+    'js/profile-deliveries': './js/app/profile/deliveries.js',
+    'js/restaurant-form': './js/app/restaurant/form.jsx',
+    'js/delivery-form': './js/app/delivery/form.jsx',
+    'js/tracking': './js/app/tracking/index.jsx',
```

Comment:
Looks nicer indeed, I wasn't sure if it was ok to have a "path" in the entry key.